libs/email-service/src/tools/design/heading.hbs

```diff
@@ -1,3 +1,10 @@
 {{#> center}}
-  <h1 class="heading">{{{copy}}}</h1>
+  {{#if eyebrow}}
+    <h2 class="heading heading-eyebrow text-{{ternary align align 'center'}}">
```

Comment:
It would be nice to add in the documentation the new props available and also about the new "list" component

Reply:
true, i'll do that

Reply:
@jeremybarbet done